src/test/java/com/easypost/WebhookTest.java

```diff
@@ -0,0 +1,113 @@
+package com.easypost;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.easypost.exception.EasyPostException;
+import com.easypost.model.Webhook;
+import com.easypost.model.WebhookCollection;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.junit.jupiter.api.MethodOrderer;
+import static org.junit.jupiter.api.Assertions.*;
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public class WebhookTest {
+    private static Map<String, Object> params = new HashMap<>();
+    private static Webhook webhook;
+
+    /**
+     * Setup the testing environment for this file.
+     *
+     * @throws EasyPostException when the request fails.
+     */
+    @BeforeAll
+    public static void setup() throws EasyPostException {
+        EasyPost.apiKey = System.getenv("EASYPOST_TEST_API_KEY");
+
+        params.put("url", "http://example.com");
+
+        webhook = Webhook.create(params);
```

Comment:
Let's go ahead and prepend `global` to each of these objects created in the `setup` function. 

Let's do this throughout the whole project's test suite.